steps/pagerduty/0.1.0/step.yml

```diff
@@ -0,0 +1,109 @@
+title: PagerDuty
+summary: |
+  Create a PagerDuty alert
+description: |
+  This step creates a PagerDuty alert with specific parameters. It can use Events API v1 or v2.
+website: https://github.com/PagerDuty/bitrise-step-pagerduty
+source_code_url: https://github.com/PagerDuty/bitrise-step-pagerduty
+support_url: https://github.com/PagerDuty/bitrise-step-pagerduty/issues
+published_at: 2019-09-16T16:15:01.059024-04:00
+source:
+  git: https://github.com/PagerDuty/bitrise-step-pagerduty.git
+  commit: f725ec05df22c26c0926e7599af05e7fc32ffae2
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: curl
+  - name: jq
+  apt_get:
+  - name: git
+  - name: curl
+  - name: jq
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- api_version: v2
+  opts:
+    description: Events API v1 or v2 (available in the Integrations tab of the Service
+      Details).
+    is_expand: false
+    is_required: true
+    summary: Events API v1 or v2 (available in the Integrations tab of the Service
+      Details)
+    title: API version
+    value_options:
+    - v1
+    - v2
+- integration_key: null
+  opts:
+    description: The hexadecimal service key (available in the Integrations tab of
+      the Service Details).
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: The hexadecimal service key (available in the Integrations tab of the
+      Service Details)
+    title: Integration key
+    value_options: []
+- event_description: null
+  opts:
+    description: The description that the event will be created with.
+    is_expand: true
+    is_required: true
+    summary: The description that the event will be created with
+    title: Event description
+    value_options: []
+- severity: error
+  opts:
+    description: The severity that the event will be created with (relevant to Event
+      Rules).
+    is_expand: false
+    is_required: false
+    summary: The severity that the event will be created with (relevant to Event Rules)
+    title: Event severity (API v2 only)
+    value_options:
+    - critical
+    - error
+    - warning
+    - info
+- skip_if_build_is_passing: "true"
```

Comment:
steps have default option `run_if` which can be used to control wether the step should run or not, pls use this option instead of a new step input: https://github.com/bitrise-io/bitrise/blob/master/_examples/experimentals/templates/bitrise.yml#L108

Reply:
@godrei where in the workflow editor can the users of my step control the `run_if`? I want them to decide if it runs on success/failure or only on failure, which is kind of the opposite of most steps, which will run on success, or maybe after a failure.

Reply:
Hi, the `run_if` property can only be set in the `bitrise.yml` mode of the Workflow Editor, there's no option to set it on the UI.

Reply:
😞 While I agree that exiting functionality should be used over new functionality, running the step only when the workflow is failing is the *default* use case for this step, and having to edit `bitrise.yml` to get the desired behaviour is not something I'd want my company's logo on.

If `run_if` is ever exposed in the UI, I'll re-submit my step, thanks.